components/data-bridge/org.wso2.carbon.databridge.core/src/main/java/org/wso2/carbon/databridge/core/DataBridge.java

```diff
@@ -437,6 +444,11 @@ public boolean deleteStreamDefinition(String streamName,
         return streamDefinitionStore.deleteStreamDefinition(streamName, streamVersion, tenantId);
     }
 
+    @Override
```

Comment:
This is due to the fact that the new method being added to StreamDefinitionStore interface and DataBridgeSubscriberService extends StreamDefinitionStore. DataBridge class itself implements DataBridgeSubscriberService. We can move this method to the AbstractStreamDefinitionStore so that this method does not come to the API itself. Will move it as such.